git-interactive-rebase-tool.1

```diff
@@ -0,0 +1,26 @@
+.\" Manpage for interactive-rebase-tool.
+.\" Contact dev@mitmaro.ca to correct errors or typos.
+.TH INTERACTIVE\-REBAS 1 "02/02/2018" "v0.5.0" "Git Manual"
```

Comment:
I think there is an `E` missing here on `REBAS`

Reply:
I'm also not sure if this should be under the Git Manual, but for now I think that's fine. :D